jsapp/js/components/modalForms/submission.es6

```diff
@@ -247,135 +263,236 @@ class Submission extends React.Component {
     const s = this.state.submission;
     let translationOptions = this.state.translationOptions;
 
-    return (
-      <bem.FormModal>
-        {this.state.promptRefresh &&
-          <div className='submission--warning'>
-            <p>{t('Click on the button below to load the most recent data for this submission. ')}</p>
-            <a onClick={this.triggerRefresh} className='kobo-button kobo-button--blue'>
-              {t('Refresh submission')}
-            </a>
-          </div>
-        }
-
-        {this.props.asset.deployment__active &&
+    // Use this modal if we just duplicated a submission, but not if we are
+    // editing it
+    if (this.state.isDuplicated && !this.state.isEditingDuplicate) {
+      return(
+        <bem.FormModal>
+          <h1 className='submission-duplicate__header'>{t('Duplicate created')}</h1>
+          <p className='submission-duplicate__text'>
+            {t('A duplicate of the submission record was successfully created. You can view the new instance below and make changes using the action buttons below.')}
+            <br/>
+            <br/>
+            {t('Source submission uuid: ')}
```

Comment:
A nitpick - I think it would be a little bit better as `{t('Source submission uuid:') + ' '}`